driver/csiplugin/controllerserver.go

```diff
@@ -2169,12 +2168,23 @@ func (cs *ScaleControllerServer) DeleteVolume(newctx context.Context, req *csi.D
 			}
 		}
 
+		// Check if fileset exists and the creator is IBM Storage Scale CSI driver
+		filesetInfo, err := conn.ListFileset(ctx, FilesystemName, FilesetName)
+		loggerId := utils.GetLoggerId(ctx)
+		if err != nil {
+			klog.Errorf("[%s]  unable to list fileset [%v] in filesystem [%v]. Error: %v", loggerId, FilesetName, FilesystemName, err)
+			return nil, status.Error(codes.Internal, fmt.Sprintf("unable to list fileset [%v] in filesystem [%v]. Error: %v", FilesetName, FilesystemName, err))
+		} else if !reflect.ValueOf(filesetInfo).IsZero() && filesetInfo.Config.Comment != connectors.FilesetComment {
+			return nil, status.Error(codes.Internal, fmt.Sprintf("Fileset [%v] is not created by IBM Container Storage Interface driver", FilesetName))
```

Comment:
We should succeed the deletion by logging the message something like this
- Skipping the fileset deletion since fileset is not created by CSI and returning success. 

Otherwise delete volume will keep coming back and user will not have any clue